### ISSUE TITLE
Playing audio to scream device breaks steamvr tracking

### ISSUE THREAD

**Issue by icewind1991** [duncanthrax/scream#109]
Whenever audio is playing to the scream device, steamvr tracking becomes completely unusable, constantly losing connection to the tracked devices, and the devices flying all over the place. If no audio is playing to the scream device tracking works perfectly.

I've reproduced this on 2 separate windows installations with the same result.

**Comment by duncanthrax** [comment]
Is this on real hardware or a VM?

**Comment by icewind1991** [comment]
On a VM, with a GPU and pcie usb controller passed trough

**Comment by duncanthrax** [comment]
And the VR HW is on the USB that is passed through? Do you use Scream in Network or IVSHMEM mode?

**Comment by icewind1991** [comment]
Yes, the vr is connected to the passed trough usb controller, I'm using scream in network mode.

**Comment by duncanthrax** [comment]
Experiment around networking and USB settings.
My first try would be to select a different emulation HW model for your virtual NIC.

Or move Scream to use passthrough via IVSHMEM too.
